sdks/python/apache_beam/io/fileio.py

```diff
@@ -649,7 +649,7 @@ class _CompressedFile(object):
   def __init__(self,
                fileobj,
                compression_type=CompressionTypes.GZIP,
-               read_size=16384):
+               read_size=gcsio.DEFAULT_READ_BUFFER_SIZE):
```

Comment:
Please clearly mention the increase of buffer size in PR description (16 KB to 16 MB).

Most other references to gcsio seems to be in ChannelFactory which is fine. Only other problematic reference I see is gcsio.MAX_BATCH_OPERATION_SIZE which probably should be removed at some point as well. I would like not to add more such references if possible :)